pkg/kn/flags/podspec_helper_test.go

```diff
@@ -35,15 +35,149 @@ func getPodSpec() (*corev1.PodSpec, *corev1.Container) {
 
 func TestUpdateEnvVarsNew(t *testing.T) {
 	spec, _ := getPodSpec()
-	env := []corev1.EnvVar{
+	expected := []corev1.EnvVar{
 		{Name: "a", Value: "foo"},
 		{Name: "b", Value: "bar"},
 	}
-	found, err := util.EnvToMap(env)
+	argsEnv := []string{
+		"a=foo",
+		"b=bar",
+	}
```

Comment:
@zroubalik could please add a test case that uses reverse order (or some alphabetical mix), that'd verify the issue with kn always ordering by key.

Reply:
Done: https://github.com/knative/client/pull/1328/commits/97466727ac5bd37ce9a08e18602827fd4d4b6858#diff-88a7bb094597d334c604ecc29d3780cc31646e892d4517b3bc8c833eb833bac8R54